tests/test_daz_ftz/test.sh

```diff
@@ -37,7 +37,7 @@ export VFC_BACKENDS_SILENT_LOAD="TRUE"
 rm -f run_parallel
 for REALTYPE in "float" "double"; do
     compile $REALTYPE
-    for BACKEND in "libinterflop_mca.so" "libinterflop_bitmask.so"; do
+    for BACKEND in "libinterflop_mca.so"; do # "libinterflop_bitmask.so"; do
```

Comment:
Why do you remove bitmask here ?

Reply:
It was a mistake, thanks for pointing it out!